src/main/java/org/opentripplanner/standalone/config/buildconfig/NetexConfig.java

```diff
@@ -158,6 +158,29 @@ private static NetexFeedParameters.Builder mapFilePatternParameters(
           .docDefaultValue(dft.ferryIdsNotAllowedForBicycle())
           .asStringSet(base.ferryIdsNotAllowedForBicycle())
       )
+      .addRouteToCentroidStopPlaceIds(
+        config
+          .of("routeToCentroidStopPlaceIds")
+          .since(V2_6)
+          .summary("List stop places that should route to centroid.")
+          .description(
+            """
+            This field contains a list of stop place ids for which the centroid will be used instead
+            of the stop coordinates.
+            
+            When doing street routing from/to a station the default behaviour is to route to any of
+            the stop place child quays. This can cause strange results for stop places that have quays
+            spread over a large area.
+            
+            For some stop places you might instead wish to use the centroid of the stop place as the
+            source/destination for street routing. In this case the centroid will be  used both for
+            direct street search and for access/egress street search where the stop place is used as
+            the start/end of the access/egress.
+            """
```

Comment:
```suggestion
            """
            This field contains a list of station ids for which the centroid will be used instead
            of the stops in routing. This apply to all routing to and from these stations including 
            direct street routing, flex and transit.
            
            When doing street routing from/to a station the default behaviour is to route to any of
            the stop place child quays. This can cause strange results for stop places that have quays
            spread over a large area. For some stations you might instead wish to use the centroid of
            the stop place as the origin/destination. Be aware that with this feature enabled, you will
            get a waking leg from stops inside the station to the centroid.
            """
```

Reply:
Actually, this is not really accurate. You won't get any walking legs for stops that belong to the station. Se my comment below with a screenshot to illustrate.

Reply:
What do you think about this text instead?
```
        This field contains a list of station ids for which street legs will start/end at the
        station centroid instead of the child stops.
        
        When searching from/to a station the default behaviour is to route from/to any of the
        stations child stops. This can cause strange results for stations that have stops spread
        over a large area. For some stations you might instead wish to use the centroid of the
        station as the origin/destination.
        
        In this case the centroid will be used both for direct street search and for access/egress
        street search where the station is used as the start/end of the access/egress. But transit
        that starts/ends at the station will work as usual without any additional street leg from/to
        the centroid.
```